kotlin-ktor/src/main/java/com/baeldung/graphql/server/api/ConferencePublisher.kt

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.graphql.server.api
+
+import com.baeldung.graphql.server.data.Conference
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+
+object ConferencePublisher {
+
+    private val mutableConferenceFlow: MutableSharedFlow<Conference> = MutableSharedFlow(replay = 1)
```

Comment:
What is the significance of the `replay = 1` parameter?

Reply:
When a new collector subscribes to the flow, he will be emitted the last `replay` values emitted. I would leave a comment but I think this is described well in the `MutableSharedFlow` docs which are easily accessible from IDE's by navigating to them